webplugin/js/app/components/rating-service.js

```diff
@@ -0,0 +1,78 @@
+class RatingService {
+    constructor() {
+        this.appOptions = {};
+        this.isCsatRatingBase = 0;
+    }
+    init(appOptions = {}) {
+        const { widgetSettings = {} } = appOptions;
+        this.appOptions = appOptions;
+        this.isCsatRatingBase = widgetSettings.csatRatingBase
+            ? widgetSettings.csatRatingBase
+            : 3;
+        if (this.isCsatRatingBase == 5) {
+            this.starCsatHtmlEnable();
+        }
+    }
+    starCsatHtmlEnable = () => {
+        $applozic('.mck-smilies-inner').removeClass('vis').addClass('n-vis');
```

Comment:
can we use direct javascript function and not use $applozic as we will be removing $applozic (jquery) usage at some point in order to reduce chat widget size
there is some existing function for add and remove class, that can be used